### PR TITLE
FORMS-304, FORMS-526 page link crash and sizing 

### PR DESCRIPTION
**Pull request by jenbutongit**
# Description

- fixed issue causing the app to crash when links are clicked
- fix type guard  

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce
the testing if necessary.

Before PR's can be merged they will need to be tested by QA and approved where
applicable. To flag the change to QA assign **@XGovFormBuilder/qa** as one of the reviewers.

- unit
- manual 

https://user-images.githubusercontent.com/22080510/113904117-9e994e80-97c9-11eb-94a3-b64044a08679.mov

# Checklist:

- [x] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and versioning
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto master and there are no code conflicts
- [x] I have checked deployments are working in all environments
- [x] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

File: runner/src/server/plugins/engine/components/ListFormComponent.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export class ListFormComponent extends FormComponent {
   options: ListComponentsDef["options"];
 
   get items(): Item[] {
+    console.log("list", this.list);
```
Comment (kakanisridhar) [inline runner/src/server/plugins/engine/components/ListFormComponent.ts:17]: clean up

**Review by kakanisridhar — DISMISSED** [review]
LGTM

**Comment by superafroman** [comment]
Damn, I've tackled these issues as well - this is the problem with not using one issue tracker :)

Also, from what I coudl see `values` is never used, so maybe better to remove?

**Review by TheSpartan1980 — APPROVED** [review]
Both changes tested locally, the issues have now been resolved @jenbutongit

**Pull request merged**